.github/workflows/laravel-package.yml

```diff
@@ -0,0 +1,25 @@
+name: Testing Laravel Package
+
+on: [push, pull_request]
+
+jobs:
+  laravel:
+    name: Laravel Package (PHP ${{ matrix.php-versions }} on ${{ matrix.operating-system }})
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-latest]
+        php-versions: ['7.2']
```

Comment:
This allows us to specify more operating systems, and more PHP versions and it will create a separate check for each of them.

For example, we could do:

```
matrix:
   operating-system: [ubuntu-latest, windows-latest, macOS-latest]
   php-versions: ['7.2', '7.3']
```

And it will literally create the check for each operating system on each PHP version and will run checks and tests upon each of them. 

It would look like this:

_Don't mind failing tests, as they were failing for testing purposes._
![image](https://user-images.githubusercontent.com/15839776/70261737-5f04c180-1793-11ea-8409-147862023c14.png)